code/src/components/InGame.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components'
+
+import { nextStep } from 'reducers/game'
+
+import Summary from './Summary'
+import Loading from './Loading'
+
+const InGameContainer = styled.section`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  margin: 30px;
+
+`
+const ActionContainerWrapper = styled.div`
+  @media (min-width: 900px) {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+} 
+`
+
+const ActionContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  background: radial-gradient(circle, rgba(15,103,72,1) 0%, rgba(5,50,16,1) 100%);
+  padding: 10px;
+  margin-bottom: 20px;
+  border-radius: 5px;
+
+  @media (min-width: 900px) {
+    width: 30%;
+  } 
+`
+
+const MainContent = styled.div`
+  height: 100vh;
+  background: black;
+  margin-top: 30px;
+
+  @media (max-width: 332px) {
+    font-size: 12px;
+  }
+`
+
+const Button = styled.button`
+    margin-top:5px;
```

Comment:
This is a bit inconsistent, sometimes you have space after colon and sometimes you don't